solidity/contracts/converter/BancorConverter.sol

```diff
@@ -593,19 +586,18 @@ contract BancorConverter is IBancorConverter, TokenHandler, SmartTokenController
       * 
       * @param _reserveToken    reserve token contract address
       * @param _sellAmount      amount of tokens to sell (in units of the smart token)
-      * @param _minReturn       if the conversion results in an amount smaller the minimum return - it is cancelled, must be nonzero
       * @param _beneficiary     wallet to receive the conversion result
       * 
       * @return amount of tokens received (in units of the reserve token)
     */
-    function sell(IERC20Token _reserveToken, uint256 _sellAmount, uint256 _minReturn, address _beneficiary) internal returns (uint256) {
+    function sell(IERC20Token _reserveToken, uint256 _sellAmount, address _beneficiary) internal returns (uint256) {
         // ensure that the input amount was already deposited
         require(_sellAmount <= token.balanceOf(this));
 
         (uint256 amount, uint256 feeAmount) = getSaleReturn(_reserveToken, _sellAmount);
 
-        // ensure the trade gives something in return and meets the minimum requested amount
-        require(amount != 0 && amount >= _minReturn);
+        // ensure the trade gives something in return
+        require(amount != 0);

```

Comment:
@yudilevi : Doesn't this mean that we don't actually need to verify minReturn > 0 everywhere?